plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -203,29 +157,31 @@ class NewCreateServiceModal extends Component {
   }
 
   onMarathonStoreServiceCreateError(errors) {
-    this.setState({errors, isPending: false});
+    this.setState({
+      apiErrors: MarathonErrorUtil.parseErrors(errors),
```

Comment:
Wonder why we need to save the errors to the state as `apiErrors` instead of parsing and pushing them to the general error list directly?